survey-gizmo-ruby.gemspec

```diff
@@ -18,8 +18,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'activesupport', '>= 3.0'
   gem.add_dependency 'addressable', '>= 2'
   gem.add_dependency 'awesome_print', '>= 1'
-  gem.add_dependency 'faraday', '>= 0.9.1'
-  gem.add_dependency 'faraday_middleware'
+  gem.add_dependency 'faraday', '>= 2.0'
```

Comment:
could you remove ruby 2.4 and 2.5 from CI as faraday >2 requires at least tuby 2.6 and any way those version are already EOL

Reply:
Removed `2.4` and `2.5` 
Also updated gemspec to set min ruby version to `>=2.6`

All <3 Ruby is EOL, would you like further adjustments to remove all `2.x` versions?